CONTRIBUTING.md

```diff
@@ -60,7 +60,7 @@ Currently, these are the combinations of system dependencies that work for MacOS
 
 ```bash
 node -v
-v16.14.2
+v18.0.0
```

Comment:
I was hoping to stick to the latest LTS version of node - which is 16.14.2 - is it possible to try that ? If not, then Im happy to move over to v18 - but, we will then need to change (in a separate PR) all our CI scripts, etc to move to v18

Reply:
will resolve based on #19